docker/run_worker_queue.sh

```diff
@@ -14,6 +14,10 @@ if test ! -d /var/lib/acme/live/"$QUEUE_ADDRESS"; then
     acmetool want "$QUEUE_ADDRESS"
 fi
 
```

Comment:
Might be a good idea to add the "don't steamroll forward if a command fails" (`e`) and "crash on undefined variables (`u`) safeguards here. (Not earlier, since there indeed are undefined variables...)

```suggestion
set -eu
```